eve-tools/Makefile

```diff
@@ -11,15 +11,18 @@ DESTDIR = /usr/
 PREFIX = local
 
 eve_run: libevetools.so
-	$(CC) tools/eve_run.cpp cpp/api.pb.cc $(CPPFLAGS) -o eve_run $(LDLIBS)
+	$(CC) tools/eve_run.cpp proto/api.pb.cc $(CPPFLAGS) -o eve_run $(LDLIBS)
 
-libevetools.so: lib/src/sendrecv.cpp
-	$(CC) $(CPPFLAGS) -shared -fPIC -o libevetools.so lib/src/sendrecv.cpp lib/src/eve_tpm_service.cpp cpp/api.pb.cc -lprotobuf
+protoc_files:
+	protoc --cpp_out=proto -Iproto api.proto
```

Comment:
Should we document in README.md that protoc needs to be installed to build this?

Reply:
It is listed in the [INSTALL.md](https://github.com/lf-edge/eve-tools/blob/master/INSTALL.md#1-install-the-prerequisite-libraries), the `protobuf-compiler`.